pageserver/src/virtual_file/owned_buffers_io/write.rs

```diff
@@ -219,119 +237,105 @@ impl Buffer for BytesMut {
     }
 }
```

Comment:
This impl is unused, can be removed.

Reply:
9f384a8426c3faf6b6f6bfa71d59343a0935b3ad: remove unused.